src/app/builder/builder.component.html

```diff
@@ -107,9 +111,16 @@
         </fieldset>
 
         <fieldset>
-          <img [src]="props.imageUrl || defaults.imageUrl" id="artwork-preview">
-          <label>Artwork URL</label>
-          <input type="text" [(ngModel)]="props.imageUrl" id="imageUrl" [placeholder]="defaults.imageUrl" name="imageUrl">
+          <img [src]="props.epImageUrl || defaults.epImageUrl" id="artwork-preview">
+          <label>Episode Artwork URL</label>
+          <p><small>This artwork will be featured prominently in the foreground of the player.</small></p>
+          <input type="text" [(ngModel)]="props.epImageUrl" id="epImageUrl" [placeholder]="defaults.epImageUrl" name="epImageUrl">
+        </fieldset>
+
+        <fieldset>
+          <label>Feed Artwork URL</label>
```

Comment:
I think I'm maybe confused about this too. My understanding is that we are using the blurred feed artwork as the default background to match the episode image. Maybe if they don't like how that looks, then they can override. What if they aren't making an episode player but are making a play the most recent episode player, was there any need to have these overrides?